### PR TITLE
Remove metrics file

### LINKED ISSUE #2642: Remove file metrics exporter and enable http metrics exporter by default
To only have a single metrics solution we should get rid of the custom file metrics exporter. And expose the metrics over HTTP by default.

### PR DESCRIPTION
**Pull request by menski**
closes #2642

### CODE REVIEW

File: docs/src/operations/metrics.md [hunk 1]
```diff
@@ -29,44 +29,23 @@ Metrics can be configured in the [configuration file](operations/the-zeebecfgtom
 
 ## Connecting Prometheus
 
-As explained, Zeebe writes metrics to a file. The default location of the file is `$ZB_HOME/metrics/zeebe.prom`. There are two ways to connect Zeebe to Prometheus:
+As explained, Zeebe exposes the metrics over a HTTP server. The default port is `26000`.
```
Comment (deepthidevaki) [inline docs/src/operations/metrics.md]: Is this port different from MetricsCfg.DEFAULT_PORT (=9600)?

Comment (menski) [reply docs/src/operations/metrics.md]: Sorry my fault. I mixed it up with our other ports :D I will fix it

**Comment by menski** [comment]
I force pushed the fix

File: docs/src/operations/metrics.md [hunk 2]
```diff
@@ -29,44 +29,23 @@ Metrics can be configured in the [configuration file](operations/the-zeebecfgtom
...
-- job_name: zb
+- job_name: zeebe
   scrape_interval: 15s
-  metrics_path: /zeebe.prom
+  metrics_path: /metrics
   scheme: http
   static_configs:
   - targets:
-    - localhost:8000
+    - localhost:26000
```
Comment (deepthidevaki) [inline docs/src/operations/metrics.md]:
```suggestion
    - localhost:9600
```

**Review by deepthidevaki — CHANGES_REQUESTED** [review]
:+1: Just one minor fix to the doc

**Comment by menski** [comment]
I pushed the fix, my apologies for my sloppiness :see_no_evil:

**Review by deepthidevaki — APPROVED** [review]
:+1:

**Pull request merged**